ignite3/src/test/java/site/ycsb/db/ignite3/IgniteAbstractClientTest.java

```diff
@@ -19,32 +19,56 @@ public class IgniteAbstractClientTest {
   public static Properties properties() {
     Properties result = new Properties();
 
-    result.put("hosts","192.168.209.148");
+    result.put("hosts", "192.168.209.148");
 
     return result;
   }
 
   @Test
   public void testCreateDdl() throws DBException {
-    doTestCreateDdl(
-        "CREATE ZONE IF NOT EXISTS Z1 WITH STORAGE_PROFILES='default,myColumnarStore';",
-        List.of(CREATE_TABLE_DDL +
-                " ZONE \"Z1\" STORAGE PROFILE 'default' SECONDARY ZONE \"Z1\" SECONDARY STORAGE PROFILE 'myColumnarStore'"),
-        true);
+    Properties properties = properties();
+
     doTestCreateDdl(
         "",
         List.of(CREATE_TABLE_DDL),
-        false);
+        properties);
+  }
+
+  @Test
+  public void testCreateDdlWithColumnar() throws DBException {
+    Properties properties = properties();
+
+    properties.put("useColumnar", String.valueOf(true));
+
+    doTestCreateDdl(
+        "CREATE ZONE IF NOT EXISTS Z1 STORAGE PROFILES ['default,myColumnarStore'];",
```

Comment:
This should be updated according to my previous comment.
```suggestion
        "CREATE ZONE IF NOT EXISTS Z1 STORAGE PROFILES ['default', 'myColumnarStore'];",
```